content/applications/finance/fiscal_localizations/argentina.rst

```diff
@@ -872,6 +872,103 @@ The menu also displays critical information related to these operations, such as
 - The :guilabel:`Journal` in which the check is currently registered
 - The **partner** associated with the operation (either customer or vendor).
 
+Ecommerce Electronic Invoicing
```

Comment:
Add custom anchor to link to this section from the **Modules installation** section, and update heading casing to sentence case
```suggestion
.. _argentina/ecommerce-electronic-invoicing:

Ecommerce electronic invoicing
```